tests/playwright-test/expect-to-pass.spec.ts

```diff
@@ -260,3 +260,35 @@ test('should give priority to timeout parameter over timeout in config file', as
   4 |         await test.expect(() => {
   `.trim());
 });
+
+test('should respect intervals in config file when intervals parameter is not passed', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.js': `module.exports = { expect: { toPass: { intervals: [100, 200] } } }`,
+    'a.spec.ts': `
+      import { test, expect } from '@playwright/test';
+      test('should fail', async () => {
+        await test.expect(() => {
+          expect(1).toBe(2);
+        }).toPass();
+      });
+    `
+  });
+  expect(result.exitCode).toBe('TODO');
+  // TODO
```

Comment:
For testing the functionality, one possible option is to configure toPass with `{ timeout: 2000, intervals: [100, 1000] }`, have a test like the following:

```js
      test('should fail', async () => {
        let attempt = 0;
        await test.expect(() => {
          expect(++attempt).toBe(-1);
        }).toPass();
      });
```
 and then check that the output contained something like this:
```
        Error: expect(received).toBe(expected) // Object.is equality·
        Expected: -1
        Received: 3·
```